<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="user_tab">
       <attribute name="title">
        <string>USER</string>
       </attribute>
       <widget class="QLabel" name="osLabel1">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>33</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>ostypes</string>
        </property>
       </widget>
       <widget class="QLabel" name="osLabel2">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>56</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>version</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>56</y>
          <width>78</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>OS Version:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>74</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Hostname:</string>
        </property>
       </widget>
       <widget class="QLabel" name="userLabel">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>9</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>hostname</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>33</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>OS Types:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="powerOffButton">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>320</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Power Off</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Start time:</string>
        </property>
       </widget>
       <widget class="QLabel" name="startTimeLabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>221</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Running time:</string>
        </property>
       </widget>
       <widget class="QLabel" name="runningTimeLabel">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>211</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="process_tab">
       <attribute name="title">
        <string>Process</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="cpu_tab">
       <attribute name="title">
        <string>CPU</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>CPU:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="cpuNameLabel">
          <property name="text">
           <string>cpu info</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>CPU Mhz:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="cpuMhz">
          <property name="text">
           <string>number</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Cache Size:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="cpuCacheSizeLabel">
          <property name="text">
           <string>number</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>CPU used:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="cpuUsed">
          <property name="text">
           <string>used</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mem_tab">
       <attribute name="title">
        <string>Memory</string>
       </attribute>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>mem total:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>mem free:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>mem usage:</string>
        </property>
       </widget>
       <widget class="QLabel" name="memTotalLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>10</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
       <widget class="QLabel" name="memFreeLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
       <widget class="QLabel" name="memUsageLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>mem available:</string>
        </property>
       </widget>
       <widget class="QLabel" name="memAvailableLabel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="memProgressBar">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>541</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
